library/bignum_mod_raw.h

```diff
@@ -0,0 +1,79 @@
+/**
+ *  Low level modular bignum functions
```

Comment:
Low-level is normally hyphenated
```suggestion
 *  Low-level modular bignum functions
```